docs/developers/Centaur.md

```diff
@@ -2,7 +2,12 @@ Centaur is an integration testing suite for the [Cromwell](http://github.com/bro
 
 ## Prerequisites
 
-Centaur expects to find a Cromwell server properly configured and running in server mode, listening on port 8000.  This can be configured by modifying the `cromwellUrl` parameter in `application.conf`.
+Centaur expects to find a Cromwell server properly configured and running in server mode, listening on port 8000.  
+This can be configured by modifying the `cromwellUrl` parameter in `application.conf`.
+
+You can get a build of your current cromwell code with [these instructions](Building.md).
+The server can be run with `java -jar <Cromwell JAR> server`. 
```

Comment:
Perhaps a general instruction to run the Cromwell server and a link to the existing documentation in `../CommandLine.md`?